<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>241</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QComboBox" name="cbOp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>添加</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>删除</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>修改</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数字查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>正则查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>前缀查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自动校正</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="leInput">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbCount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExec">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwResult">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>241</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>239</width>
       <height>86</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnKey1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnKey2">
        <property name="text">
         <string>2（ABC）</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnKey3">
        <property name="text">
         <string>3（DEF）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnKey4">
        <property name="text">
         <string>4（GHI）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnKey5">
        <property name="text">
         <string>5（JKL）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnKey6">
        <property name="text">
         <string>6（MNO）</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnKey7">
        <property name="text">
         <string>7（PQRS）</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnKey8">
        <property name="text">
         <string>8（TUV）</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnKey9">
        <property name="text">
         <string>9（WXYZ）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
